docker-compose.yml

```diff
@@ -1,10 +1,17 @@
 services:
   db:
     image: postgres:13.14
+    ports:
+      - "5432:5433"
```

Comment:
Can we keep original PG as is?

Reply:
It will conflict with the default db port. I'm just following the previous PR.

Reply:
Can't we keep PostgreSQL at 5432 (as is default) and run Timescale at 5433 or so?

Reply:
Let's double check if it's possible. Never did it.